db/src/db.rs

```diff
@@ -191,6 +197,9 @@ fn get_user_version(conn: &rusqlite::Connection) -> Result<i32> {
 
 // TODO: rename "SQL" functions to align with "datoms" functions.
 pub fn create_current_version(conn: &mut rusqlite::Connection) -> Result<DB> {
+    if rusqlite::version_number() < MIN_SQLITE_VERSION {
```

Comment:
This should be in `ensure_current_version`, down below — `create_current_version` doesn't run when simply opening an existing database.